mixins/storable.es

```diff
@@ -0,0 +1,16 @@
+const Storable = Element =>
+
+class extends Element {
+
+  retrieve (records, id=this.tagName) {
+    records = JSON.stringify (records)
+
+    this.context = JSON.parse
+      (localStorage.getItem (id) || records)
+  }
+
+  store (id=this.tagName) {
+    localStorage.setItem
+      (id, JSON.stringify (this.context))
```

Comment:
this seems like a pretty naive approach but no worries there, given the use case. I wonder if it could be named something clearer, like `SimpleStorableStateElement` or whatever to clarify purpose/intent a bit more.